$OpenBSD: patch-Python_modsupport_c,v 1.3 2003/12/30 17:05:30 sturm Exp $

        This has been submitted to the Python team as part of patch
        554716.

--- Python/modsupport.c.orig	2001-08-17 20:39:25.000000000 +0200
+++ Python/modsupport.c	2003-12-29 21:50:32.000000000 +0100
@@ -411,8 +411,12 @@ Py_VaBuildValue(char *format, va_list va
 #ifdef VA_LIST_IS_ARRAY
 	memcpy(lva, va, sizeof(va_list));
 #else
+#ifdef __va_copy
+	__va_copy(lva, va);
+#else
 	lva = va;
 #endif
+#endif
 
 	if (n < 0)
 		return NULL;
